misc/nydfs-bitlicense-lawsky-update.md

```diff
@@ -7,6 +7,9 @@ date: 2014-12-18
 
 NY DFS BitLicense update 2014-12-18
 
+<http://bipartisanpolicy.org/events/payment-policy-in-the-21st-century-the-promise-of-innovation-and-the-challenge-of-regulation/>
```

Comment:
http://web.archive.org/web/20150511110453/http://bipartisanpolicy.org/events/payment-policy-in-the-21st-century-the-promise-of-innovation-and-the-challenge-of-regulation/